src/cbapi/legacy/cbapi.py

```diff
@@ -294,6 +294,14 @@ def sensor(self, sensor_id):
         r.raise_for_status()
         return r.json()
 
+    def sensor_resourcestatus(self, sensor_id):
+        """
+        get resourve utilization for a single sensor, as specified by sensor id
+        """
+        r = self.cbapi_get("%s/api/v1/sensor/%s/resourcestatus" % (self.server, sensor_id))
+        r.raise_for_status()
+        return r.text
```

Comment:
is the output of the resourcestatus endpoint JSON? If so let's decode it from json here. I get 204 ("no content") on my cb server when calling this endpoint against any of my sensors.